src/transport/UTransport.cpp

```diff
@@ -10,3 +10,61 @@
 // SPDX-License-Identifier: Apache-2.0
 
 #include "up-cpp/transport/UTransport.h"
+#include "up-cpp/datamodel/validator/UUri.h"
+#include "up-cpp/datamodel/validator/UMessage.h"
+
+using namespace uprotocol;
+using namespace uprotocol::transport;
+namespace UriValidator = uprotocol::datamodel::validator::uri;
+namespace MessageValidator = uprotocol::datamodel::validator::message;
+
+UTransport::UTransport(const v1::UUri& defaultSrc)
+    : defaultSource_(defaultSrc) {
+
+    auto [bOk, reason] = UriValidator::isValidRpcResponse(defaultSource_);
```

Comment:
The PR should be rebased to the latest up-cpp main. A new validator has been added for this check to remove ambiguity: `isValidDefaultSource()`

Reply:
Done